### DISCUSSION TITLE
Anyone want free 100$ credit for digitalocean

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#657] [(A) Anything]
I dont have a credit card and i only have the student promo code so just comment if u want it

**Comment by ghost** [comment]
uhhhh okay

**Comment by irhrhd** [comment]
Okay how do you want me to send u the code

**Reply by ghost** [reply]
dck.dachickenking@gmail.com

**Reply by irhrhd** [reply]
Sent

**Reply by irhrhd** [reply]
Also of your a student you can sign up too <a href=https://education.github.com/pack/>this is where you sign up for the pack</a>

**Reply by irhrhd** [reply]
Did you get it?

**Reply by irhrhd** [reply]
I like :octocat:
